console/src/main/kotlin/app/accrescent/parcelo/console/routes/auth/GitHub.kt

```diff
@@ -43,6 +49,23 @@ fun AuthenticationConfig.github(
                 clientId = clientId,
                 clientSecret = clientSecret,
                 defaultScopes = listOf("user:email"),
+                onStateCreated = { call, state ->
+                    // Cross-site request forgery (CSRF) protection.
+                    // See https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-30#section-10.12
+                    val environment = call.application.environment
+                    val developmentMode = environment.developmentMode
```

Comment:
nit: Probably unnecessary to create this extra variable, and not having it is more consistent with `environment.oauthStateCookieName`